DANS-KNAW/easy-split-multi-deposit
# EASY 1011 - Add DepositorID

fixes EASY-1011
#### When applied it will
- require a depositorID to be added to input CSV file
- check that there is one and only one unique depositorID given per dataset
- verify that this user is an _active_ user within Ldap
- add the depositorID to the `deposit.properties` as `depositor.userId=<depositorID>`
#### Where should the reviewer @DANS-KNAW/easy start?

Start in `AddPropertiesToDeposit.scala` which contains all the logic for this change
#### How should this be manually tested?
- Create a folder `target/springfield`
- Run `./run.sh -s target/springfield target/test-classes/Roundtrip_MD/sip-demo-2015-02-24 target/result`
- This should produce a set of deposits in `target/result`
- Every deposit should have a `deposit.properties` containing amongst others `depositor.userId=<depositorID>`
#### related pull requests on github

| repo | PR |
| --- | --- |
| easy-dtap | [PR 5](https://github.com/DANS-KNAW/easy-dtap/pull/5) |
